### ISSUE TITLE
Tutorial - Perhaps change placement of Fury usage.

### ISSUE THREAD

**Issue by lcenine** [TerryCavanagh/diceydungeonsbeta#351]
It gives the impression you _have_ to use Fury at the start of the round before you do anything else.

Perhaps instead of a 6, have the Warrior roll a 3, and make him re-roll before having Fury pointed out and used.

That would then be reaffirmed against Hothead in the tutorial.

**Comment by ncrecc** [comment]
Even that could probably be done better - the notion that skillcards don't get furied might be hard to follow at first

**Comment by TerryCavanagh** [comment]
Hmm, I don't think I've seen any confusion about this while testing the game at shows, and changing the tutorial is a real pain. I'm going to leave this as is - thanks for the suggestion, though 👍
